bcgov/PIMS
# PIMS-1925 Project Editing Admins Only

## 🎯 Summary

  
[PIMS-1925](https://citz-imb.atlassian.net/browse/PIMS-1925)


## Changes
- Disabled edit buttons on ProjectDetails if lacking the Admin role
- Added check in update projects controller to look for Admin role first
## Testing
- Make sure you can still edit projects as an admin
- Make sure as a general user you can't see the edit options


## 🔰 Checklist


> - I have performed a self-review of my code.
> - I have commented my code, particularly in hard-to-understand areas.
> - I have made corresponding changes to the documentation where required.
> - I have tested my changes to the best of my ability.
> - My changes generate no new warnings.
